CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 1.2.0
+
+* Add support for HTML challenges 
+* Add support for invisible Device Check challenge
+* Add support for additional challenges following an invisible Device Check
```

Comment:
We should remove this line.
It's the expected behavior for invisible Device Check, so it's weird to put a dedicated item for it.

It made sense for other SDKs only because support for invisible Device Check was released in a broken state.